features/product/viewing_products/seeing_correct_lowest_prices_when_period_changes_on_the_channel.feature

```diff
@@ -0,0 +1,52 @@
+@viewing_products
+Feature: Seeing the correct product's lowest price when the period on the channel changes
+    In order to know whether the current discount is attractive
+    As a Guest
+    I want to see a product's lowest price within the channel's set period
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And it is "2023-01-01" now
+        And the store has a product "Wyborowa Vodka" priced at "$10.00"
+        And on "2023-01-05" its price changed to "$11.00"
+        And on "2023-01-10" its price changed to "$12.00" and original price to "$20.00"
+        And on "2023-01-15" its price changed to "$13.00" and original price to "$15.00"
+        And on "2023-01-20" its price changed to "$14.00" and original price to "$10.00"
+        And on "2023-01-25" its price changed to "$15.00" and original price to "$5.00"
+        And it is "2023-01-30" now
+
+    @todo
+    Scenario: Seeing the correct lower price from 30 days before the discount
+        Given this channel has 30 days set as the lowest price for discounted products checking period
+        When I check this product's details
+        Then I should see "$10.00" as this product's lowest price from before the discount
+
+    @todo
+    Scenario: Seeing the correct lower price from 25 days before the discount
+        Given this channel has 25 days set as the lowest price for discounted products checking period
+        When I check this product's details
+        Then I should see "$11.00" as this product's lowest price from before the discount
+
+    @todo
+    Scenario: Seeing the correct lower price from 20 days before the discount
+        Given this channel has 20 days set as the lowest price for discounted products checking period
+        When I check this product's details
+        Then I should see "$12.00" as this product's lowest price from before the discount
+
+    @todo
+    Scenario: Seeing the correct lower price from 15 days before the discount
+        Given this channel has 15 days set as the lowest price for discounted products checking period
+        When I check this product's details
+        Then I should see "$13.00" as this product's lowest price from before the discount
+
+    @todo
+    Scenario: Seeing the correct lower price from 5 days before the discount
+        Given this channel has 5 days set as the lowest price for discounted products checking period
+        When I check this product's details
+        Then I should see "$14.00" as this product's lowest price from before the discount
```

Comment:
I'm not sure if we need so many scenarios, as they all check the same thing, we can reduce the number of them slightly 😃